src/main/kotlin/io/github/mojira/arisa/modules/FutureVersionModule.kt

```diff
@@ -28,7 +28,7 @@ class FutureVersionModule(
                 addComment(CommentOptions(messagePanel))
             } else {
                 latestVersion!!.add()
-                if (resolution == null || resolution == "Unresolved") {
+                if (resolution == null || resolution == "Unresolved" || resolution == "Awaiting Response") {
```

Comment:
This is really hacky and I don't think we want to do so. Also there are cases where the ReopenAwaiting module won't reopen the ticket even if a version is added, e.g. the version is added by a volunteer.

Reply:
So what should be done to prevent both cases from happening?